jetpack/jetpack-start-endpoints/plan-cancellation.md

```diff
@@ -0,0 +1,86 @@
+# Cancelling a plan
+
+Plans can be cancelled by making a request using your partner token from the step above and the URL of the site being cancelled.
+
+### Endpoint Information (/partner-cancel)
+
+- __Method__: POST
+- __URL__:    `https://public-api.wordpress.com/rest/v1.3/jpphp/{$site}/partner-cancel`
+
+`$site` is the site's domain and path where `/` in the path is replaced with `::`. For example:
+
+| Site URL              | $site Identifier        |
+| --------------------- | -------------------     |
+| `example.com`         | `example.com`           |
+| `example.com/demo`    | `example.com::demo`     |
+| `example.com/demo/wp` | `example.com::demo::wp` |
+
+### Request Parameters (/partner-cancel)
+- __plan__: A slug representing which plan or item to cancel. One of `free`, `personal`, `premium`, or `professional` for plans. For backups either `jetpack-backup-realtime` or `jetpack-baackup-daily`. Note this is an optional parameter for Jetpack Plans (Personal, Premium, Professional) but required for cancelling standalone items including Jetpack backups.
+- __cancel-all__: Optional. If this option is set to true ALL purchased items provisioned by the calling partner for a given site URL will be cancelled.
+
+
+### Query Parameters (/partner-cancel)
+- __http_envelope__: Default to `false`. Sending `true` will force the HTTP status code to always be `200`. The JSON response is wrapped in an envelope containing the "real" HTTP status code and headers.
+- __pretty__:        Defaults to `false`. Setting to `true` will output pretty JSON.
+
+### Response Parameters (/partner-cancel)
+
+Below, the response parameters are grouped by whether the request to cancel errored or not.
+
+#### Successful response (/partner-cancel)
```

Comment:
This section needs to be updated for the `cancel-all` case where we add some extra information to the response.

Reply:
Added a section for cancel-all to the response documentaiton

Reply:
Considering that the response would look something like this when `cancel-all` is set in the request:

```
{
    "pending_activations": [],
    "activations": [],
    "success": true
}
```

This is what I would suggest:

>
> - __success__:       (bool) Was the operation successful?. It is possible for success to be false if a plan did not exist for the site. Note: If `cancel-all` was set in the request, then the value will be `true` only if ALL items for the site were successfully cancelled. 
>
> - __pending_activations__: (conditional) (array) If `cancel-all` if set for the request, then this property will be added to the response to indicated all pending activations that were cancelled.
>
> - __activations__: (conditional) (array) If `cancel-all` if set for the request, then this property will be added to the response to indicated all activations that were cancelled.